addon/components/search-result/template.hbs

```diff
@@ -33,7 +33,9 @@
             </div>
 
             {{!Last edited on on}}
-            <div class="m-t-sm"> <em> {{t "eosf.components.searchResult.lastEdited"}}: {{moment-format result.date_updated 'YYYY-MM-DD'}} </em> </div>
+            {{#if result.date_updated}}
```

Comment:
There are SHARE results - this was adapted from [the preprints search results component](https://github.com/CenterForOpenScience/ember-preprints/blob/develop/app/templates/components/search-result.hbs#L28)